.circleci/config.yml

```diff
@@ -159,9 +161,107 @@ jobs:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
 
+  # Run all integration tests for all packages with `--integration` flag
+  # Tested against Chrome and Firefox on Mac and Windows
+  integration_tests:
+    executor: main-executor
+    environment:
+      COVERAGE: true
+    steps:
+      - checkout_and_fetch
+      - restore_node_modules
+      - restore_built_packages
+      # Install JQ for parsing the JSON response from GitHub api
+      - jq/install
+      # This script is to see if a PR is approved or not before running the integration tests
+      # It curls the Pull Requests data from GitHub's api and uses JQ to check if the status of the PR is approved
+      - run:
```

Comment:
What is the purpose of waiting for approval to run the integration tests? It should be an indicator of the PR's health and one of the things I take into account when reviewing a PR